### PR TITLE
MAUI 833923 - [Hotfix] Update Maui ImageEditor's API Links

### PR DESCRIPTION
**Pull request by devanathan-vaithiyanathan**
[Hotfix] Update Maui ImageEditor's API Links

### CODE REVIEW

File: MAUI/ImageEditor/crop.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ imageEditor.Crop(ImageCropType.Free);
 
 {% endhighlight %}
 
-After the cropping area has been selected, the `SaveEdits` method is called, which in turn crops the selected region and displays the cropped image on the image editor.
+After the cropping area has been selected, the [SaveEdits](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_SaveEdits) method is called, which in turn crops the selected region and displays the cropped image on the image editor.
 
 {% highlight C# %}
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/crop.md:40]: add link - `CancelEdits`

File: MAUI/ImageEditor/crop.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ imageEditor.Crop(ImageCropType.Circle);
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/crop.md:104]: image

Update content:

This means that no aspect ratio and the cropped image retains its original width and height ratio.

Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/crop.md:104]: image

ImageCropType.Ratio

Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/crop.md:104]: image

`ImageCropType.Circle`

File: MAUI/ImageEditor/crop.md (hunk 1 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/crop.md:40]: added

File: MAUI/ImageEditor/crop.md [hunk 3]
```diff
@@ -136,7 +136,7 @@ imageEditor.SaveEdits();
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/crop.md:136]: image

File: MAUI/ImageEditor/crop.md (hunk 2 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/crop.md:104]: added

Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/crop.md:104]: added

File: MAUI/ImageEditor/crop.md (hunk 3 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/crop.md:136]: added

File: MAUI/ImageEditor/imagefilter.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Image Effect in .NET MAUI Image Editor (SfImageEditor)
 
-Using the image editor control, apply various effects such as Brightness, Hue, Saturation, Contrast, Blur, Opacity, and Sharpen to your image. These effects can be applied using the toolbar or by utilizing the `ImageEffect` method. The `ImageEffect` method consists of two arguments: `ImageEffect` and `EffectValue`. The ImageEffect is an enumeration that includes the following effects:
+Using the image editor control, apply various effects such as Brightness, Hue, Saturation, Contrast, Blur, Opacity, and Sharpen to your image. These effects can be applied using the toolbar or by utilizing the [ImageEffect](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEffect.html) method. The [ImageEffect](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEffect.html) method consists of two arguments: `ImageEffect` and `EffectValue`. The `ImageEffect` is an enumeration that includes the following effects:
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/imagefilter.md:12]: `ImageEffect` => add enum link in 2 places

File: MAUI/ImageEditor/imagefilter.md [hunk 5]
```diff
@@ -24,7 +24,7 @@ Using the image editor control, apply various effects such as Brightness, Hue, S
 The EffectValue are the corresponding ImageEffect values, which varies for each effect, and they are explained as follows.
 
 N> The Image Effect enum also contains “None” option, which removes all the previously applied effects, which are not saved.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/imagefilter.md:26]: The Image Effect enum also contains “None” option, which removes all the previously applied effects, which are not saved.

File: MAUI/ImageEditor/imagefilter.md [hunk 6]
```diff
@@ -24,7 +24,7 @@ Using the image editor control, apply various effects such as Brightness, Hue, S
 The EffectValue are the corresponding ImageEffect values, which varies for each effect, and they are explained as follows.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/imagefilter.md:24]: The `EffectValue` are the corresponding `ImageEffect`(link) values, which vary for each effect and are explained as follows.

File: MAUI/ImageEditor/imagefilter.md (hunk 5 above)
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/imagefilter.md:26]: The `ImageEffect`(link) enum also contains `ImageEffect.None`(link) option, which removes all the previously applied effects, which are not saved.

File: MAUI/ImageEditor/imagefilter.md (hunk 4 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/imagefilter.md:12]: added

File: MAUI/ImageEditor/imagefilter.md (hunk 5 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/imagefilter.md:26]: added

File: MAUI/ImageEditor/imagefilter.md [hunk 7]
```diff
@@ -162,15 +162,15 @@ public MainPage()
 
 {% endhighlight %}
 
-The applied effects can be canceled using the `CancelEdits` method or by calling the ImageEffect method with ImageEffect.None.
+The applied effects can be canceled using the [CancelEdits](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_CancelEdits) method or by calling the ImageEffect method with ImageEffect.None.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/imagefilter.md:165]: or by calling the `ImageEffect`(method link) method with `ImageEffect.None`(link).

File: MAUI/ImageEditor/imagefilter.md (hunk 6 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/imagefilter.md:24]: added

File: MAUI/ImageEditor/transformation.md [hunk 8]
```diff
@@ -13,7 +13,7 @@ The Image Editor in .NET MAUI provides options for rotating and flipping images,
 
 ## Rotate
 
-The `Rotate` method is used to rotate images with annotations. Each rotation will rotate the image 90 degrees clockwise. You can rotate the image using either the toolbar or the `Rotate` method.
+The [Rotate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_Rotate) method is used to rotate images with annotations. Each rotation will rotate the image 90 degrees clockwise. You can rotate the image using either the toolbar or the `Rotate` method.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/transformation.md:16]: toolbar or the `Rotate`(link) method.

File: MAUI/ImageEditor/imagefilter.md (hunk 5 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/imagefilter.md:26]: added

File: MAUI/ImageEditor/transformation.md [hunk 9]
```diff
@@ -25,7 +25,7 @@ N> Angle cannot be specified in the code to alter the rotation angle of the imag
 
 ## Flip
 
-The image editor control is capable of showing the mirror image. The `Flip` method allows you to flip the image horizontally or vertically based on the `ImageFlipDirection`. Flipping the image can be done by either using a toolbar or the `Flip` method.
+The image editor control is capable of showing the mirror image. The [Flip](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_Flip_Syncfusion_Maui_ImageEditor_ImageFlipDirection_) method allows you to flip the image horizontally or vertically based on the [ImageFlipDirection](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageFlipDirection.html). Flipping the image can be done by either using a toolbar or the `Flip` method.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/transformation.md:28]: using a toolbar or the `Flip`(link) method.

File: MAUI/ImageEditor/transformation.md (hunk 8 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/transformation.md:16]: added

File: MAUI/ImageEditor/getting-started.md [hunk 10]
```diff
@@ -15,11 +15,11 @@ This section explains the steps to create and load an image to the image editor
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/getting-started.md:15]: image

This section explains the steps to create and load an image to the `.Net MAUI ImageEditor(SfImageEditor)`(https://www.syncfusion.com/maui-controls/maui-image-editor)
 control.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**